samples/dashboards/googlesheets/tutorial/demo.js

```diff
@@ -0,0 +1,194 @@
+// Google Sheets Credentials
```

Comment:
I would create a `samples/dashboards/data` directory for any data-related demos.

Reply:
The directory already exists, I'll move the demo there.